From 2a60adc35668b4bf9184742915935f91a388c81d Mon Sep 17 00:00:00 2001
From: Huiquan Zhong <huiquan.zhong@intel.com>
Date: Sat, 18 Oct 2014 00:44:10 +0800
Subject: [PATCH 043/441] dma: dw: enable CherryTrail LPDMA

Add CherryTrail's LPDMA ACPI ID(80862286, 808622C0) to the
mapping table.

Change-Id: I58cfd9cd404e612c3df745c1dad8c21cef369416
Tracked-On: https://jira01.devtools.intel.com/browse/GMINL-2493
Signed-off-by: Huiquan Zhong <huiquan.zhong@intel.com>
(cherry picked from commit 4a4b1c7431a7b972b712f04b50ddc06bbd707246)

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/acpi/acpi_lpss.c  | 4 ++--
 drivers/dma/dw/platform.c | 2 ++
 2 files changed, 4 insertions(+), 2 deletions(-)

--- a/drivers/acpi/acpi_lpss.c
+++ b/drivers/acpi/acpi_lpss.c
@@ -194,8 +194,8 @@ static const struct acpi_device_id acpi_
 	/* Cherrytrail LPSS devices */
 	{ "808622C1", (unsigned long)&cht_i2c_dev_desc },
 	{ "8086228A", (unsigned long)&byt_uart_dev_desc },
-	{ "80862286", }, /* CHT DMA1 */
-	{ "808622C0", }, /* CHT DMA2 */
+	{ "80862286", (unsigned long)&lpss_dma_desc },
+	{ "808622C0", (unsigned long)&lpss_dma_desc },
 	{ "8086228E", (unsigned long)&byt_spi_dev_desc },
 	{ "80862288", }, /* CHT PWM */
 
--- a/drivers/dma/dw/platform.c
+++ b/drivers/dma/dw/platform.c
@@ -256,6 +256,8 @@ MODULE_DEVICE_TABLE(of, dw_dma_of_id_tab
 #ifdef CONFIG_ACPI
 static const struct acpi_device_id dw_dma_acpi_id_table[] = {
 	{ "INTL9C60", 0 },
+	{ "80862286", 0 },
+	{ "808622C0", 0 },
 	{ }
 };
 MODULE_DEVICE_TABLE(acpi, dw_dma_acpi_id_table);
